frontend/lib/src/hostComm/HostCommunicationManager.tsx

```diff
@@ -189,6 +190,10 @@ export default class HostCommunicationManager {
       this.props.sendAppHeartbeat()
     }
 
+    if (message.type === "SET_APP_DISABLED") {
+      this.props.setAppDisabled(message.disabled)
+    }
```

Comment:
Curious if we should call this `widgetDisabled` instead of `appDisabled` to keep it consistent with the eventual prop?  I wonder if there's some distinction between disabling the *app* vs disabling the *widgets* that, as written, could confuse.

Reply:
Hm, yeah, I think it may be worth considering naming everything `widgetsDisabled` and similar for consistency. I'm not sure if there's any effective difference between the two aside from potentially in some obscure edge cases, but there's less room for confusion if the `widgetsDisabled` name is used

Reply:
There is one small problem with using the `widgets` name: We will soon add support for events that trigger reruns on a lot of non-widget elements (e.g. charts, expander, tabs, dataframe...). And these elements also need to be disabled. We haven't really discussed yet if we will start calling a lot more elements widgets, or if we somehow move away from this categorization. Another more generic alternative might be `rerunsDisabled`, but not sure if this is better than `appDisabled`.

Reply:
+1 to Lukas's comment. For the events like chart selections, we probably want to still allow those actions on the frontend but just disable the server trigger. `rerunsDisabled` might be the most explicit name. `inputsDisabled` is another idea.

Reply:
Thanks for the additional context and name suggestions. I like `inputsDisabled` since it conveys that reruns from widget and non-widget input will be prevented. It also suggests that a disabled UX will be displayed for these elements which is important. I'll update the PR to use this name.